P0009/P0009.md

```diff
@@ -41,6 +41,9 @@ Special thanks to Tomasz Kaminski for invaluable help in preparing this paper fo
 
 - significant wording updates based on LWG feedback 
 - `submdspan` was moved into its own paper, allowing mdspan to go forward even if there is not enough time to review `submdspan`
+- Move wording change in paragraph 1 of [mdspan.extents.comp] from P2553 to P0009
```

Comment:
The label seems to be [mdspan.extents.compare], [mdspan.extents.cmp] or [mdspan.extents.comp], depending on what I'm looking at, including a difference between P2553R1 published and P2553R1 at the head of P2553-P0009-LWG-small-group-20220531.

Please go through both documents and make sure they are consistent with the labels in currently published documents. (It's okay to change the labels, but say that you are doing so.)